lib/option.fz

```diff
@@ -131,7 +139,7 @@ is
 
   # unwraps an option if it exists, returns default value otherwise.
   #
-  public get(default Lazy T)
+  public or(default Lazy T)
```

Comment:
Should IMHO be called `or_else`, this is at least the name chosen in Java`s `Optional`.